### PR TITLE
Added level, item level, afterglow information to printed information

### PR DESCRIPTION
**Pull request by Traelblaser**

### CODE REVIEW

File: collector/collector.lua [hunk 1]
```diff
@@ -28,19 +29,20 @@ sorted_bags = L{'safe', 'safe2', 'storage', 'locker',
...
 
 
-function add_result(result,bag,count,augments)
+function add_result(result,bag,count,augments,lvl)
     local count = count > 1 and ' ('..count..')' or ''
+    local level = lvl == nil and '' or ' %s':format(lvl)
     local rank = augments ~= nil and augments.rank ~= nil and ' (Rank: %s)':format(augments.rank) or ''
-    return (bag == 'missing' and result:color(259) or result:color(258)) .. count .. rank
+    return (bag == 'missing' and result:color(259) or result:color(258)) .. count .. level .. rank
 end
```
Comment (lili-ffxi) [inline collector/collector.lua]: Not a huge deal but checking for a variable to be nil is a little roundabout, I'd turn around the level check for consistency with the other value.

File: collector/collector.lua [hunk 2]
```diff
@@ -59,8 +61,12 @@ function curate(set)
 			data = inventory[bag][i]
 			if data.id ~= 0 then
                 local name = res_items[data.id].name
+                local level = res_items[data.id].item_level or res_items[data.id].level or ''
+                local ag = item_descriptions[data.id] and item_descriptions[data.id].en:find("Afterglow") and true or false
+                level = level == 99 and ag and (level .. " II") or level == 119 and ag and (level .. " III") or level
```
Comment (lili-ffxi) [inline collector/collector.lua]: The bottom two lines needs to go under an `if` that checks if it's actually needed (i.e. `level and level == 99`, most likely). The least we touch `item_description` table the better, and due to how resources work it's not actually loaded until it's accessed the very first time.

File: collector/collector.lua [hunk 3]
```diff
@@ -10,6 +10,7 @@ require('tables')
 local slips = require('slips')
 local res_items = require('resources').items
 local key_items = require('resources').key_items
+local item_descriptions = require('resources').item_descriptions
```
Comment (lili-ffxi) [inline collector/collector.lua]: At this point there's no longer need to keep these tables separate and we might just as well have a single `res` table.

**Review by lili-ffxi — COMMENTED** [review]
Awesome. A few nitpicks. I can take care of them after accepting the PR if you don't have time for them.

File: collector/collector.lua (hunk 3 above)
Comment (Traelblaser) [reply collector/collector.lua]: Fixed in [691cb22](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/691cb22ca38feac47d38f43db13cedd8f5834ad3)

File: collector/collector.lua (hunk 2 above)
Comment (Traelblaser) [reply collector/collector.lua]: Fixed in [a84395a](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/a84395a2c2d509121138be33ee702a5be90401bc)

File: collector/collector.lua (hunk 1 above)
Comment (Traelblaser) [reply collector/collector.lua]: True.. the consistency with the rest of the code is better.  Fixed in [a84395a](https://github.com/lili-ffxi/FFXI-Addons/pull/13/commits/a84395a2c2d509121138be33ee702a5be90401bc)

**Pull request merged**